src/main/resources/assets/opencomputers/lang/de_DE.lang

```diff
@@ -117,12 +123,14 @@ item.oc.UpgradeContainerCard2.name=Karten-Behälter (Stufe 3)
 item.oc.UpgradeContainerUpgrade0.name=Upgrade-Behälter (Stufe 1)
 item.oc.UpgradeContainerUpgrade1.name=Upgrade-Behälter (Stufe 2)
 item.oc.UpgradeContainerUpgrade2.name=Upgrade-Behälter (Stufe 3)
-item.oc.UpgradeCrafting.name=Fertigungs-Upgrade
+item.oc.UpgradeCrafting.name=Crafting-Upgrade
```

Comment:
General convention sais that Crafting should be translated as "Fertigung" in automation environments (and robots are a kind of Automation).